cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -86,6 +87,11 @@ func (m command) build() *cli.Command {
 	}
 
 	c.Flags = []cli.Flag{
+		&cli.StringSliceFlag{
+			Name:        "config-search-path",
+			Usage:       "Specify the path to search for config files when discovering the entities that should be included in the CDI specification.",
```

Comment:
nit: "config files" is a bit vague here. It would be helpful if we specified what configuration files they are

Reply:
nvm, I guess cdi config files is more or less implied